relay/main.py

```diff
@@ -88,10 +108,28 @@ def _show_version(ctx, param, value):
 @click.pass_context
 def main(ctx, port, config, addresses, version):
     """run the relay server"""
-    logger.info("Starting relay server version %s", get_version())
+    logging.basicConfig(
+        format="%(asctime)s[%(levelname)s] %(name)s: %(message)s",
+        level=os.environ.get("LOGLEVEL", "INFO").upper(),
+    )
+
     # silence warnings from urllib3, see github issue 246
-    get_logger("urllib3.connectionpool", level=logging.CRITICAL)
-    trustlines = TrustlinesRelay(config_json_path=config, addresses_json_path=addresses)
+    logging.getLogger("urllib3.connectionpool").setLevel(logging.CRITICAL)
+
+    logger.info("Starting relay server version %s", get_version())
+
+    # we still handle the json file for now. This will be removed
+    # soon. But it's nice to see the end2end tests succeed.
```

Comment:
Well that did not work.

Reply:
this is fixed now. I've rebased the PR and merged some commits. It's ready for review.